bids/layout/tests/test_layout.py

```diff
@@ -479,22 +493,24 @@ def test_get_with_regex_search_bad_dtype(layout_7t_trt):
     # Two runs (1 per session) for each of subjects '10' and '01'
     assert len(results) == 4
 
-
+# AD: Changed this test to use a entity found in ds005
+# 'session' was not in dataset, therefore not suggested
```

Comment:
Isn't the point that `session=True` is invalid?

Reply:
Maybe, depends on how the above is resolved, but I thought it was just a simple test of using the wrong name for an entity (i.e. "ses" instead of "session")

Reply:
Oh, that was probably it.